### PR TITLE
Update streams.

### LINKED ISSUE #16224: Broken: SPOTV And SPOTV2 Korean
### Broken Link

https://bcovlive-a.akamaihd.net/99545824f51f4c1592b35314c6ece3b0/ap-northeast-1/6271486521001/playlist.m3u8

### What happened to the stream?

Not loading

### Notes (optional)

broken link
im trying to find other links

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #16262: Broken: Lolokoko streams
### Broken Link

All links that start with https://cdn-nvm-chicago.lolokoko.tv/

### What happened to the stream?

Other

### Notes (optional)

All lolokoko.tv streams currently showing this image with no additional context given
vlcsnap-2024-03-28-13h10m17s199

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Alstruit**
Closes #16224
Closes #16262

### CODE REVIEW

File: streams/id.m3u [hunk 1]
```diff
@@ -343,3 +343,7 @@ https://ott-balancer.tvri.go.id/live/eds/TVRIWorld/hls/TVRIWorld.m3u8
 https://streamtawaf.asfatv.com/hls/tawaftv.m3u8
 #EXTINF:-1 tvg-id="TVMu.id",TV Mu (720p) [Not 24/7]
 https://e.siar.us/live/tvmu.m3u8
+#EXTINF:-1 tvg-id="",SPOTV (720p)
+http://202.150.161.68/spotv1/index.m3u8
+#EXTINF:-1 tvg-id="",SPOTV2 (720p)
+http://202.150.161.68/spotv2/index.m3u8
```
Comment (BellezaEmporium) [inline streams/id.m3u:349]: I don't quite understand why you want to move those channels there.

image

**Review by BellezaEmporium — APPROVED** [review]

**Comment by cysjunk** [comment]
I hope someone finds more Polish channels soon.

**Review by cysjunk — COMMENTED** [review]
I hope someone will find more Polish channels soon.

File: streams/id.m3u (hunk 1 above)
Comment (Alstruit) [reply streams/id.m3u:349]: It's an Indonesian feed.

**Review by PopeyeTheSai10r — APPROVED** [review]

**Pull request merged**